test/new-e2e/system-probe/test/micro-vm-init.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eo xtrace
+
+GOVERSION=$1
+KITCHEN_DOCKERS=/kitchen-docker
+
+# Add provisioning steps here !
+## Set go version correctly
+eval $(gimme $GOVERSION)
+## Start docker
+systemctl start docker
+## Load docker images
+find $KITCHEN_DOCKERS -maxdepth 1 -type f -exec docker load -i {} \;
+
+# VM provisioning end !
+
+# Start tests
+/system-probe-test_spec
```

Comment:
I *think* you want `exec` here

```suggestion
exec /system-probe-test_spec
```